### PR TITLE
Layoutchange/responsive

### PR DESCRIPTION
**Pull request by Kratika0907**
Layout change and build error fix.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -10543,10 +10543,7 @@
     "react-infinite-scroll-component": {
       "version": "5.1.0",
       "resolved": "https://registry.npmjs.org/react-infinite-scroll-component/-/react-infinite-scroll-component-5.1.0.tgz",
-      "integrity": "sha512-ZQ7lYlLByil1ZVdvt57UJeBIj/tSOwKyds+B0LIm/xGmxzJJZwW/hYGKKs74sHO3LoIDwcq2ci6YXqrkWSalLQ==",
-      "requires": {
-        "throttle-debounce": "^2.1.0"
```
Comment (swarajpure) [inline package-lock.json:10548]: are you sure if infinite scroll works after removing `throttle-debounce`?

File: styles/Home.module.css [hunk 2]
```diff
@@ -129,20 +88,99 @@
   box-shadow: 0 0 20px -7px rgba(0, 0, 0, 0.65);
 }
 
-.content {
-  flex-shrink: 1;
+
+
+
+
```
Comment (swarajpure) [inline styles/Home.module.css:94]: Isn't that a lot of blank spaces?

File: styles/Home.module.css [hunk 3]
```diff
@@ -129,20 +88,99 @@
...
+.navBar li a:hover {
+ text-decoration: underline;
+}
+
+
+
+	.homeContainer {
+		display: grid;
+    gap: 10px;
```
Comment (swarajpure) [inline styles/Home.module.css:132]: Not properly indented.

**Review by swarajpure — COMMENTED** [review]
Have some doubts with CSS and package-lock.json, please clarify. Thank you! 😄

File: styles/Home.module.css (hunk 3 above)
Comment (ankushdharkar) [reply styles/Home.module.css:132]: Great catches @swarajpure

File: components/coins-status/index.js [hunk 4]
```diff
@@ -47,10 +46,8 @@ const CoinsStatus = (props) => {
           .coins-container {
             display: flex;
             flex-direction: row;
-            justify-content: space-evenly;
-            align-self: flex-end;
-            width: 33%;
-            flex-wrap: nowrap;
+            //justify-content: space-evenly;
+            // align-self: flex-end;
```
[the remark this answers is not in the document]
Comment (ankushdharkar) [reply components/coins-status/index.js:50]: Let's not keep dead code around. Great point @swarajpure

File: styles/Home.module.css [hunk 5]
```diff
@@ -129,20 +88,99 @@
...
+
+	
+
+	.transactionGraph {
+		grid-row: 1/2;
+   
+	}
+.transactionMenu {
+		grid-row: 2/3 ;
+
```
Comment (ankushdharkar) [inline styles/Home.module.css:181]: Not properly formatted

**Review by ankushdharkar — CHANGES_REQUESTED** [review]

File: components/transaction-chart/index.js [hunk 6]
```diff
@@ -46,7 +46,7 @@ const TransactionChart = (props) => {
 
   useEffect(() => {
     chart();
-  }, []);
+  }, [chart]);
```
Comment (Harshi7016) [inline components/transaction-chart/index.js:50]: I will update this changes after the merge

File: styles/Home.module.css (hunk 3 above)
Comment (Harshi7016) [reply styles/Home.module.css:92]: @Kratika0907 I was in the thought it gets automatically indented on save. It is a good idea to have that right?

**Review by swarajpure — APPROVED** [review]

**Review by Harshi7016 — APPROVED** [review]

**Pull request merged**